src/main/java/it/gov/pagopa/pu/workflow/service/exportfile/ExportFileService.java

```diff
@@ -1,7 +1,9 @@
 package it.gov.pagopa.pu.workflow.service.exportfile;
 
+import it.gov.pagopa.pu.processexecutions.dto.generated.ExportFile;
 import it.gov.pagopa.pu.workflow.dto.generated.WorkflowCreatedDTO;
 
 public interface ExportFileService {
   WorkflowCreatedDTO expireExportFile(Long exportFileId);
+  WorkflowCreatedDTO create(Long exportFileId, ExportFile.ExportFileTypeEnum exportFileType);
```

Comment:
I don't like this name, too generic
```suggestion
  WorkflowCreatedDTO exportData(Long exportFileId, ExportFile.ExportFileTypeEnum exportFileType);
```
or `exportFile`, as the name of the operation on the openApi